solid/solid-file-client/index.html

```diff
@@ -37,8 +37,8 @@ <h2>Your tasks</h2>
 
         <script src="https://cdn.jsdelivr.net/npm/@inrupt/solid-client-authn-browser@1.11.2/dist/solid-client-authn.bundle.js"></script>
         <script src="https://cdn.jsdelivr.net/npm/solid-file-client@1.2.5/dist/window/solid-file-client.bundle.js"></script>
-        <script src="solid.js"></script>
-        <script src="main.js"></script>
+        <script src="./solid.js"></script>
```

Comment:
Was this change necessary? I'd rather use the same style in all examples.

Reply:
removed the change